react/src/Calendar/Calendar.stories.tsx

```diff
@@ -48,3 +48,15 @@ export const SizeSmall = CalendarOnlyTemplate.bind({})
 SizeSmall.args = {
   size: 'sm',
 }
+
+export const CalendarWithDefaultFocusedDate = CalendarOnlyTemplate.bind({})
+CalendarWithDefaultFocusedDate.args = {
+  defaultFocusedDate: new Date('2010-01-01'),
+}
+
+export const CalendarWithDefaultFocusedDateOverriddenByDefaultValue =
+  CalendarOnlyTemplate.bind({})
+CalendarWithDefaultFocusedDateOverriddenByDefaultValue.args = {
+  defaultFocusedDate: new Date('2010-01-01'),
+  defaultValue: new Date('2001-01-01'),
```

Comment:
for the test to be accurate, the defaultValue should be different from the focused date prop.

Reply:
it is different